script/update_marriage_abroad_opposite_sex_outcomes.rb

```diff
@@ -0,0 +1,130 @@
+require "csv"
+require "find"
+
+# Variations
+# 1 - commission text with link
+# 2 - embassy text with link
+# 3 - government text
+# 4 - commission text with link (above ##Prove you're free to get married)
+# 5 - embassy text with link (above ##Prove you're free to get married)
+# 6 - embassy text with link (above ##What you need)
+# 7 - consulate text with link (above ##What you need to do)
+
+COUNTRIES_DIR = "lib/smart_answer_flows/marriage-abroad/outcomes/countries/".freeze
+OPPOSITE_SEX_FILE = "_opposite_sex.govspeak.erb".freeze
+CSV_HEADINGS = %w[country variation embassy_link updated].freeze
```

Comment:
Thanks @ChrisBAshton , we have updated this now :)

> This looks like a chunky bit of content work to get done - nice one!
> 
> Are you also going to be running the script and committing the changes as part of this PR? (If so, the update_marriage_abroad_opposite_sex_outcomes.rb file can be removed at the end of the PR, so its contents aren't really that important to review... what's more important is that we check it's made the changes correctly) 😊

So we thought it would make sense to get this merged first, and then run the script and commit the updates in a separate PR. We debated on whether it was worth even keeping the script, but these changes might be removed and/or the script might be useful for any bulk updates in future.

Alternatively it could just stay on a branch and not get merged into master, we're open to suggestions!

Reply:
I'd be tempted to run the script and commit it in this PR, so that we can see it works (I don't see any particular benefit to making two separate PRs 🤔 )

And I'd still be tempted to remove the script in a commit at the end of the PR - it exists in version control so we can always go back and grab it if needed. This feels like a highly bespoke bit of scripting that doesn't have much 'general' use and will serve as a confusing "why is this here?" moment for developers poking around the codebase in the future. But there may be some related Platform Health cards I don't know about...? :)

Reply:
Yep that's a fair point, we'll go ahead and run the script and add the changes as commits to this PR 👍